### ISSUE TITLE
next/image - sizerSvg style not overriding padding, margins etc to match imgStyle

### ISSUE THREAD

**Issue by michrome** [vercel/next.js#19817]
If you target `img` margins with something like the CSS below, you end up with a stretched image.

```css
/* @tailwindcss/typography does this */
.prose img {
  margin-top: 2em;
  margin-bottom: 2em;
}
```

I _think_ this is because [`margin` and `padding` aren't being overridden for the sizerSvg](https://github.com/vercel/next.js/blob/fddad89cbccd74d74553986c79f387d2369e6185/packages/next/client/image.tsx#L384). They are being overridden for the [actual Img](https://github.com/vercel/next.js/blob/fddad89cbccd74d74553986c79f387d2369e6185/packages/next/client/image.tsx#L258), so the sizerSvg should be updated to match that.

![Screenshot 2020-12-04 at 11 27 26](https://user-images.githubusercontent.com/596770/101158898-7257a080-3624-11eb-9672-712f8aac4790.png)

**Comment by AjitZero** [comment]
Did you try using the layout prop?

I'm also using Tailwind and faced a similar issue (not under prose) which was resolved with `layout="responsive"`

**Comment by michrome** [comment]
Thanks, @AjitZero! Confirming that the issue I'm describing **only** affects the default (intrinsic) layout type.

It doesn't affect the fixed or responsive layout types, so a workaround until it's resolved may be to use `layout="responsive"`. ☑️

**Comment by michrome** [comment]
Confirmed fixed in [v10.0.5-canary.4](https://github.com/vercel/next.js/releases/tag/v10.0.5-canary.4) 🎉

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
